src/LibraryManager.Vsix/Commands/ManageLibrariesCommand.cs

```diff
@@ -32,6 +32,13 @@ public static void Initialize(Package package, OleMenuCommandService commandServ
             Instance = new ManageLibrariesCommand(package, commandService);
         }
 
+        private void BeforeQueryStatus(object sender, EventArgs e)
+        {
+            var button = (OleMenuCommand)sender;
+
+            button.Visible = true;
```

Comment:
It's still going to be invisible if not in a web project though, right? It will call into QueryStatus only if visibility constrains are satisfied otherwise?

Reply:
Yes, that's correct